### PR TITLE
Standardized primes test cases

### LINKED ISSUE #4: Standardize primes test cases

### PR DESCRIPTION
**Pull request by naslundx**
Fixes #4 

The ranges in Lua and Python are off by a factor of 10, because they take too long otherwise. We need to fix that somehow.

### CODE REVIEW

**Comment by naslundx** [comment]
Otherwise they find all primes up to 1 000 000.
Maybe it should not be allowed to assume that the number of primes < 79000.
And I think it should do something else do, like find the last digit of the sum or something.

Thoughts @dekespo ?

File: lua/primes.lua [hunk 1]
```diff
@@ -1,5 +1,6 @@
-local n = 10000
+local n = 100000 -- incorrect
 local primes = {2}
+
 for i = 3, n do
 	local isPrime = true
 	for j = 1, #primes do
```
Comment (dekespo) [inline lua/primes.lua:6]: Lua doesn't check primes[j] \* primes[j] > i . This should speed up a bit.

File: python3/primes.py [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-            return False
-    else:
-        return True
-
-result = filter(is_prime, range(1, 20000))
-
+primes = [2]
+for i in range(3, 100000): # Incorrect range!
+    is_prime = True
+    for j in range(0, len(primes)):
```
Comment (dekespo) [inline python3/primes.py:4]: maybe it's better to write as "for prime in primes"?

File: python3/primes.py [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+        p = primes[j]
+        if j*j > i:
```
Comment (dekespo) [inline python3/primes.py:6]: p (prime) or j?

Comment (naslundx) [inline python3/primes.py:6]: Oops, thanks

File: python3/primes.py (hunk 2 above)
Comment (naslundx) [inline python3/primes.py:4]: I don't remember if `for ... in ...` is guaranteed to traverse the list in order, do you know?

File: lua/primes.lua (hunk 1 above)
Comment (naslundx) [inline lua/primes.lua:6]: How do I write that in lua? :)

File: python3/primes.py (hunk 2 above)
Comment (dekespo) [inline python3/primes.py:4]: Yes, it will traverse the list in order if you append the list, which it does already.

File: lua/primes.lua (hunk 1 above)
Comment (dekespo) [inline lua/primes.lua:6]: I think it's

`if primes[j] * primes[j] > i then
    break
end`

:)

**Comment by dekespo** [comment]
🎉

**Pull request merged**